horace_core/sqw/@DnDBase/parse_args_.m

```diff
@@ -6,27 +6,30 @@
 % - args.dnd_obj   % DnD class instance
 % - args.sqw_obj   % SQW class instance
 % - args.data_struct % generic struct, presumed to represent DnD
+persistent parser
```

Comment:
this may be often. I believe once-processed parser is faster then creating and redefining it for each call. Have not profiled it yet, but all previous experience says its faster then constructing object from scratch.